### PR TITLE
Add French language

### PR DESCRIPTION
**Pull request by nabil-rejeb**

### CODE REVIEW

File: ui/src/app/locale/locale.constant-ru_RU.json [hunk 1]
```diff
@@ -1354,6 +1354,7 @@
         "language": "Язык",
         "locales": {
             "en_US": "Английский",
+			"fr_FR": "французский",
```
Comment (mp-loki) [inline ui/src/app/locale/locale.constant-ru_RU.json]: This should start with capital 'Ф'

Comment (nabil-rejeb) [reply ui/src/app/locale/locale.constant-ru_RU.json]: Thanks @mp-loki , I changed it and added missing comma in locale.constant-zh_CN.json

**Comment by ashvayka** [comment]
Hi @nabil-rejeb Please separate localization from other modifications to rule nodes in this PR. I will merge only localization for now.

**Comment by nabil-rejeb** [comment]
Hi @ashvayka ,
i made a new PR #1011 for localization only.
Thank's.

**Pull request closed without merging**